docs/examples/src/bin/serialisation.rs

```diff
@@ -27,9 +28,10 @@ where
         Ok(())
     }
 }
+
```

Comment:
You commit added a lot of new lines in this file. Can you check that all the tutorial pages that reference them still reference the right line ranges? It's a bit sensitive to this, because I didn't use the named ranged as often as I probably should have.